src/scripts/worldmap/MapHelper.ts

```diff
@@ -59,9 +72,10 @@ class MapHelper {
     public static moveToTown(townName: string) {
         if (MapHelper.accessToTown(townName)) {
             Game.gameState(GameConstants.GameState.idle);
-            $("[data-route='" + player.route() + "']").removeClass('currentRoute').addClass('unlockedRoute');
+            $("[data-route='" + player.route() + "']").removeClass('currentRoute').addClass('unlockedRoute'); //pretty sure any jquery in typescript does not work fyi
```

Comment:
Why do you think it does not work?

Reply:
when you console log out the jquery selector inside of any location in typescript - nothing gets pulled. I think the css classes are coming from the function calculateRouteCssClass instead of here.

Reply:
Yeah this line does nothing I think, you can remove it. But everywhere else jquery should work, just tested it out